easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
```

Comment:
@boegel: Ok, so once more. I wanted to say that should we update the easyblock, if the newer versisons work perfectly with  `CMakeMake`. The easyblock originally needed only due to a non-standard prefix-path option for `configure`